tests/handlers/test_koji.py

```diff
@@ -48,6 +48,143 @@ def test_build_complete(mock_koji_cs, mock_getBuild_two):
     build.commit.is_connected(commit)
 
 
+@mock.patch('koji.ClientSession')
+def test_build_building(mock_koji_cs, mock_getBuild_three):
+    """Test the Koji handler when it recieves a new build building message."""
+    mock_koji_session = mock.Mock()
+    mock_koji_session.getBuild.return_value = mock_getBuild_three
+    mock_koji_cs.return_value = mock_koji_session
+
+    with open(path.join(message_dir, 'koji', 'build_building.json'), 'r') as f:
+        msg = json.load(f)
+    assert KojiHandler.can_handle(msg) is True
+    handler = KojiHandler(config)
+    handler.handle(msg)
+
+    build = KojiBuild.nodes.get_or_none(id_='754479')
+    assert build is not None
+    assert build.completion_time == datetime(2018, 6, 15, 20, 26, 38, tzinfo=pytz.utc)
+    assert build.creation_time == datetime(2018, 8, 20, 15, 7, 27, tzinfo=pytz.utc)
+    assert build.epoch is None
+    assert build.extra == ('{"source": {"original_url": "git://pkgs.domain.com/rpms/squashfs-'
+                           'tools#09d40c9bdfd34c5130f8f02e49e059efd33bddf7"}}')
+    assert build.id_ == '754479'
+    assert build.name == 'squashfs-tools'
+    assert build.release == '17.el8'
+    assert build.version == '4.3'
+    assert build.start_time == datetime(2018, 8, 20, 15, 7, 27, tzinfo=pytz.utc)
+    assert build.state == 0
```

Comment:
The build that already existed in Neo4j was already in the building state so this test doesn't actually test that the state changed. Could you please fix that?

Reply:
NVM, I misread the code.